apps/generator/docs/generator-template.md

```
@@ -91,14 +111,16 @@ components:
 ├── template
-│   └── index.js
+│ └── index.js
 ├── test
-│   └── fixtures
-│       └── asyncapi.yml
+│ └── fixtures
+│ └── asyncapi.yml
 └── package.json
-```
+
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)